src/main/java/com/TableTalk/Enterprise/dto/Game.java

```diff
@@ -1,20 +1,17 @@
 package com.TableTalk.Enterprise.dto;
 
-
-
 import com.google.gson.annotations.SerializedName;
 import lombok.Data;
-
 import java.util.List;
 import java.util.Map;
 
 public @Data class Game {
-    @SerializedName("id")
+    @SerializedName("game_id")
     private String id;
-    @SerializedName("url")
-    private String url;
-    @SerializedName("name")
-    private String name;
+    @SerializedName("game_url")
+    private String gameUrl;
+    @SerializedName("game_name")
+    private String gameName;
```

Comment:
A couple of cautions here:

1) @SerializedName: this likely matches up to a JSON Data feed, so should not be changed.
2) No need for the game prefix on variables, as that is given, by the name of the class itself (Game).

Reply:
This makes sense now that you've explained it. Thank you.